test/src/test/java/org/corfudb/integration/SourceForwardingDataSender.java

```diff
@@ -287,6 +308,17 @@ private LogReplicationEntryMsg changeMsgMetadata(LogReplicationEntryMsg message)
         return newMessage;
     }
 
+    public void checkStatusOnStandby(boolean expectedDataConsistent) {
+        String destinationClusterId = LogReplicationIT.REMOTE_CLUSTER_ID;
+        LogReplicationMetadata.ReplicationStatusKey standbyClusterId = LogReplicationMetadata.ReplicationStatusKey.newBuilder()
+                .setClusterId(destinationClusterId)
+                .build();
+        try (TxnContext txn = standbyCorfuStore.txn(LogReplicationMetadataManager.NAMESPACE)) {
+            LogReplicationMetadata.ReplicationStatusVal standbyStatus = (LogReplicationMetadata.ReplicationStatusVal)txn.getRecord(REPLICATION_STATUS_TABLE, standbyClusterId).getPayload();
+            assertThat(standbyStatus.getDataConsistent()).isEqualTo(expectedDataConsistent);
+        }
+    }
+
```

Comment:
I saw the fact that hmm this class is only used for `LogReplicationIT` for now, but I think this method will make this class bond with one single test class. Is it possible to decouple the remote cluster id here?

Reply:
good point Chris. Shama probably you can make it part of TestConfig and it's cleaner.

Reply:
Addressed! Made it part of testConfig.